src/utilities/gql.ts

```diff
@@ -3436,7 +3487,7 @@ const gql = {
   `,
 
   UPDATE_SCHEDULING_GOAL_PLAN_SPECIFICATIONS: `#graphql
-    mutation UpdateSchedulingGoalPlanSpecifications($goalSpecsToUpdate: [scheduling_specification_goals_insert_input!]!, $goalSpecIdsToDelete: [Int!]! = [], $specificationId: Int!) {
+    mutation UpdateSchedulingGoalPlanSpecifications($goalSpecsToUpdate: [scheduling_specification_goals_insert_input!]!, $goalSpecIdsToDelete: [Int!]! = []) {
```

Comment:
Because the pkey was changed, this query is no longer "correct". Can we change `updateSchedulingGoalPlanSpecifications` to `insertSchedulingGoalPlanSpecifications`, and remove the `on_conflict` specification? The effect function using this query will also need to update its reference to `updateSchedulingGoalPlanSpecifications`